package mvp;

message OrbitalImageFileDesc {
  required string camera_path = 1;
  required string image_path = 2;
  required int32 image_cols = 3;
  required int32 image_rows = 4;
  required double radius_min = 5;
  required double radius_max = 6; 
}
